app/components/object-select/component.js

```diff
@@ -1,12 +1,18 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-  actions: {
+ actions: {
     update: function() {
+      let selected;
       const selectedEl = this.$('select')[0];
       const selectedIndex = selectedEl.selectedIndex;
       const items = Ember.A(this.get('items'));
-      const selected = items.objectAt(selectedIndex);
+
+      if (this.get('prompt')){
+        selected = items.objectAt(selectedIndex - 1);
```

Comment:
Can you abstract this a little bit?

Something like:

    let promptOffset = this.get('prompt') ? 1 : 0;
    selected = items.objectAt(selectedIndex - promptOffset);